src/popup/observe-media.js

```diff
@@ -0,0 +1,5 @@
+export default function observe(specifier, fn) {
```

Comment:
Not sure whether we can control the menubar icon, as it's statically specified in the manifest as far as I remember.

Reply:
One cannot control it from "within" the extension, it requires some extra JS listeners in the `content/background.js`. I just meant if we already have some JS for changing this in place, then having the listeners for the icon isn't too far of either (see e.g. [this post](https://stackoverflow.com/questions/58880234/toggle-chrome-extension-icon-based-on-light-or-dark-mode-browser)).

But let's not open this can of worms here.

Reply:
Uff, yeah. Maybe another time 😀